styles.css

```diff
@@ -123,6 +123,11 @@ body {
   flex-direction: row;
 }
 
+.program-card:hover {
+  border: 4px solid aquamarine;
+  cursor: pointer;
+}
+
```

Comment:
- [ ] Kindly remove the commented code that we have in line 5 - line 10 of your styles.css according to best practises. 👍